src/reducers/filter.js

```diff
@@ -0,0 +1,14 @@
+const filterState = {
+  filter: '',
+};
+
+const filterBookReducer = (state = filterState, action) => {
+  switch (action.type) {
+    case 'CHANGE_FILTER':
+      return { ...state, filter: action.payload };
+    default:
+      return state;
+  }
+};
```

Comment:
- [ ] As said in the instructions, the filter state should defaults to 'All'
![Screenshot from 2021-03-09 12-29-47](https://user-images.githubusercontent.com/30977845/110495321-2872d080-80d3-11eb-81bc-f727b38137f4.png)